apps/console/src/home.jsp

```diff
@@ -172,7 +172,11 @@
         <!-- End of custom stylesheets -->
 
         <!-- Start of custom scripts added to the head -->
-        <script type="text/javascript" src="<%= htmlWebpackPlugin.options.publicPath %>extensions/head-script.js"></script>
+        <script
+            id="head-script"
+            type="text/javascript"
+            src="<%= htmlWebpackPlugin.options.publicPath %>extensions/head-script.js"
```

Comment:
Do we need a `data-page-id` here?

Reply:
Added via: 26f2cf37e27f61532fc661befa2b7b35c734fd70
This page's `data-page-id` is not used as of now. From the script, it is handled `data-page-id` being null.